src/portable/synopsys/dwc2/dwc2_bcm.h

```diff
@@ -32,28 +32,30 @@
 #endif
 
 #include "broadcom/interrupts.h"
-//#include "broadcom/caches.h"
+#include "broadcom/caches.h"
 
 #define DWC2_REG_BASE       0xFE980000UL
 #define DWC2_EP_MAX         8
 #define DWC2_EP_FIFO_SIZE   4096
 
-//#define dcache_clean(_addr, _size)              data_clean(_addr, _size)
-//#define dcache_invalidate(_addr, _size)         data_invalidate(_addr, _size)
-//#define dcache_clean_invalidate(_addr, _size)   data_clean_and_invalidate(_addr, _size)
+#define dcache_clean(_addr, _size)              data_clean(_addr, _size)
+#define dcache_invalidate(_addr, _size)         data_invalidate(_addr, _size)
+#define dcache_clean_invalidate(_addr, _size)   data_clean_and_invalidate(_addr, _size)
 
 TU_ATTR_ALWAYS_INLINE
 static inline void dwc2_dcd_int_enable(uint8_t rhport)
 {
   (void) rhport;
   BP_EnableIRQ(USB_IRQn);
+  __asm__ volatile("isb");
```

Comment:
mentioned in https://github.com/hathach/tinyusb/pull/1163#issuecomment-960473264